test/test_run.py

```diff
@@ -246,13 +245,15 @@ def run_it():
                   if fn != 'machine.json']
 
     data = util.load_json(result_fn)
-    value = dict(zip(data['result_columns'], data['results']['time_examples.TimeSuite.time_example_benchmark_1']))
+    value = dict(zip(data['result_columns'], data['results']
+                 ['time_examples.TimeSuite.time_example_benchmark_1']))
```

Comment:
This is not very clear, can you keep `data['results']['time_examples.TimeSuite.time_example_benchmark_1']` all in the same line please.